cibuildwheel/windows.py

```diff
@@ -58,6 +57,32 @@ def simple_shell(args, env=None, cwd=None):
         args = ['cmd', '/E:ON', '/V:ON', '/C'] + args
         return subprocess.check_call(' '.join(args), env=env, cwd=cwd)
 
+    def extract_zip(zip_src, dest):
+        with ZipFile(zip_src) as zip:
+            zip.extractall(dest)
+    
+    def install_cpython(version, arch):
```

Comment:
Remove it from function body to increase readability?

Reply:
Ah, forgot this one.